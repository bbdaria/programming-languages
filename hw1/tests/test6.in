ADD
REG
1001
Introduction to Programming
3
ADD
LAB
1008
Somthing complicated
Prof. H
Lidov
ADD
SEM
1100
Advanced Topics in advanced material
09:00
11:00
PRINT
1001
PRINT
1008
PRINT
1100
END