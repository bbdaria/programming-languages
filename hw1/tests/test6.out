REG
1001
Introduction to Programming
3
LAB
1008
Somthing complicated
Prof. H
Lidov
SEM
1100
Advanced Topics in advanced material
09:00
11:00
